.travis.yml

```diff
@@ -32,6 +34,11 @@ cache:
 
 # try running against postgres 9.3
 addons:
+  apt:
+    packages:
+      - mysql-server-5.6
+      - mysql-client-core-5.6
+      - mysql-client-5.6
```

Comment:
@cebe It's related. I updated PR's first post. Please see "Integration with Travis CI" section.